addons/vests/CfgWeapons.hpp

```diff
@@ -143,4 +143,29 @@ class CfgWeapons {
         picture = QPATHTOF(UI\vest_tactical_darkblack_ca.paa);
         hiddenSelectionsTextures[] = {QPATHTOF(data\vest_tactical_darkblack_co.paa)};
     };
+
+    // Modular Carrier
+    class V_CarrierRigKBT_01_EAF_F;
+    class CLASS(Vest_Modular_Black): V_CarrierRigKBT_01_EAF_F {
+        dlc = QUOTE(PREFIX);
+        scope = 2;
+        author = "Trenchgun, Tyrone";
+        displayName = CSTRING(Vest_Modular_Black);
+        picture = QPATHTOF(UI\vest_modular_black_ca.paa);
+        hiddenSelectionsTextures[] = {QPATHTOF(data\vest_modular_black_co.paa)};
+    };
+
+    class CLASS(Vest_Modular_Green): CLASS(Vest_Modular_Black) {
+        scope = 2;
+        displayName = CSTRING(Vest_Modular_Green);
+//      picture = QPATHTOF(UI\);
```

Comment:
Missing?

Reply:
Waiting on Garrett, i'd make it myself but it'd look incredibly out of place.